### PR TITLE
doc: Thank You PhoneGap

### PR DESCRIPTION
**Pull request by erisu**

### CODE REVIEW

File: www/_posts/2020-08-14-goodbye-phonegape.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Goodbye PhoneGap
+
+[Adobe] recently [announced][AdobeBlogUpdate] that [PhoneGap] is shutting down. How is Cordova affected? In short: not much. 
+
+**Rest assured, Apache Cordova is still active and maintained!**
+
+We want to take this opportunity to thank Adobe and the PhoneGap team for all the blood, sweat, and tears they have put forward into designing and building these tools, services, and community. Without them, hybrid mobile apps would not be the same.
+
+Today, we are in the fortunate position that Cordova is an Open Source Software, which is entirely maintained by the community. PhoneGap originally started as an open-source project by Nitobi Software in 2008. Nitobi Software was acquired by Adobe in 2011 and shortly after donated PhoneGap to the [Apache Software Foundation] (ASF) as Cordova. Apache Cordova, which was a core tool of Adobe PhoneGap and PhoneGap Build for the past nine years, will continue to thrive as long as the health of the community remains strong and contributions are made. If you use, have used, or looking to start using Cordova, also consider contributing and taking part in this awesome community to help it grow.
```
Comment (raphinesse) [inline www/_posts/2020-08-14-goodbye-phonegape.md:18]: I think the following sentence has a bug :wink:

> Nitobi Software was acquired by Adobe in 2011 and shortly after donated PhoneGap to the Apache Software Foundation as Cordova.

Comment (erisu) [reply www/_posts/2020-08-14-goodbye-phonegape.md:18]: What is the bug?

Comment (erisu) [reply www/_posts/2020-08-14-goodbye-phonegape.md]: I went ahead and just deleted `(ASF)`

Comment (raphinesse) [reply www/_posts/2020-08-14-goodbye-phonegape.md]: I guess I just read it wrong. I think I get how the sentence is meant to be read now. Sorry for the bother.

**Pull request merged**